pom.xml

```diff
@@ -93,8 +96,20 @@
     <resolverVersion>1.0.0.v20140518</resolverVersion>
     <wagonVersion>3.5.1</wagonVersion>
     <surefire.version>3.0.0-M6</surefire.version>
+    <slf4j.version>1.7.36</slf4j.version>
+    <junit.version>4.13.2</junit.version>
     <maven.site.path>maven-release-archives/maven-release-LATEST</maven.site.path>
     <project.build.outputTimestamp>2022-01-02T10:18:49Z</project.build.outputTimestamp>
+
+    <maven-compiler-plugin.version>3.10.1</maven-compiler-plugin.version>
+    <maven-artifact-transfer.version>0.13.1</maven-artifact-transfer.version>
+    <maven-invoker-plugin.version>3.2.2</maven-invoker-plugin.version>
+
+    <plexus-component-metadata.version>1.7.1</plexus-component-metadata.version>
+    <plexus-maven-plugin.version>1.3.8</plexus-maven-plugin.version>
+
+    <maven-plugin-testing-harness.version>3.3.0</maven-plugin-testing-harness.version>
```

Comment:
We tend to use camel case for those kind of properties, see here: https://github.com/apache/maven-scm/blob/b04525fa0716e6de7a65bf9970284efa0e49b177/pom.xml#L128